apps/preview/nuxt/pages/showcases/MegaMenu/MegaMenuNavigation.vue

```diff
@@ -0,0 +1,606 @@
+            class="block md:hidden mr-5 bg-transparent hover:bg-primary-800 hover:text-white active:bg-primary-900 active:text-white"
+            @click="openMenu([])"
+          >
+            <SfIconMenu class="text-white" />
+          </SfButton>
+          <a
+            href="/"
+            aria-label="SF Homepage"
+            class="flex items-center mr-2 text-white md:mr-10 focus-visible:outline focus-visible:outline-offset focus-visible:rounded-sm"
+          >
+            <picture>
+              <source :srcset="brandLogo" media="(min-width: 1024px)" />
+              <img :src="brandLogoSign" alt="Sf Logo" class="w-8 h-8 lg:w-[12.5rem] lg:h-[1.75rem]" />
+            </picture>
+          </a>
+        </div>
+        <div class="flex flex-nowrap">
+          <SfButton
+            v-for="actionItem in actionItems"
+            :key="actionItem.ariaLabel"
+            :aria-label="actionItem.ariaLabel"
+            class="ml-2 text-white bg-transparent hover:bg-primary-800 hover:text-white active:bg-primary-900 active:text-white"
+            variant="tertiary"
+            square
+          >
+            <template #prefix>
+              <Component :is="actionItem.icon" />
+            </template>
+            <p v-if="actionItem.role === 'login'" class="hidden md:inline-flex">{{ actionItem.label }}</p>
+          </SfButton>
+        </div>
+      </div>
+
+      <nav ref="floatingRef">
+        <ul
+          class="hidden md:flex px-6 py-2 bg-white border-b border-b-neutral-200 border-b-solid"
+          @blur="
+            (event) => {
+              if (!(event.currentTarget as Element).contains((event.relatedTarget as Element))) {
+                close();
+              }
+            }
+          "
+        >
+          <li v-for="menuNode in content.children" :key="menuNode.key">
+            <SfButton
+              variant="tertiary"
+              class="mr-2"
+              @mouseenter="openMenu([menuNode.key])"
+              @click="openMenu([menuNode.key])"
+            >
+              <span>{{ menuNode.value.label }}</span>
+              <SfIconChevronRight class="rotate-90" />
+            </SfButton>
+
+            <div
+              v-if="isOpen && activeNode.length === 1 && activeNode[0] === menuNode.key"
+              :key="activeMenu.key"
+              :style="style"
+              class="hidden md:grid gap-x-6 grid-cols-4 bg-white shadow-lg p-6 left-0 right-0"
+              @mouseleave="close()"
+            >
+              <template v-for="node in activeMenu.children" :key="node.key">
+                <SfListItem
+                  v-if="node.isLeaf"
+                  tag="a"
+                  size="sm"
+                  :href="node.value.link"
+                  class="typography-text-sm mb-2"
+                >
+                  {{ node.value.label }}
+                </SfListItem>
+                <div v-if="node.isLeaf" class="col-start-2 col-end-5" />
```

Comment:
I guess this can follow react implementation:
```suggestion
                <template v-if="node.isLeaf">
                  <SfListItem
                    tag="a"
                    size="sm"
                    :href="node.value.link"
                    class="typography-text-sm mb-2"
                  >
                    {{ node.value.label }}
                  </SfListItem>
                  <div v-if="node.isLeaf" class="col-start-2 col-end-5" />
                </template>
```

Reply:
[`a64ac53`](https://github.com/vuestorefront/storefront-ui/pull/2640/commits/a64ac53e9a9a87e6546c8646f4ba90b122ee0ad4)